automatic/microsoft-windows-terminal/update.ps1

```diff
@@ -1,25 +1,28 @@
 Import-Module au
 
-$releases = 'https://github.com/microsoft/terminal/releases'
+$releases = 'https://api.github.com/repos/microsoft/terminal/releases' # should variable name change?: releases_page instead of releases
 
 function global:au_BeforeUpdate { Get-RemoteFiles -NoSuffix -Purge }
 
 function global:au_GetLatest {
-    $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
-
-    #Microsoft.WindowsTerminal_0.2.1831.0_8wekyb3d8bbwe.msixbundle
-    $re  = "Microsoft.WindowsTerminal_(.+)_.*.msixbundle"
-    $url = $download_page.links | Where-Object href -match $re | Select-Object -First 1 -expand href
-
-    $version = ([regex]::Match($url,$re)).Captures.Groups[1].value
-    $url = 'https://github.com' + $url
-
-    return @{
-        URL32 = $url
-        Version = $version
-        RemoteVersion  = $version
-        FileType = 'msixbundle'
+  $download_page = Invoke-RestMethod -Uri $releases # should variable name change?: releases instead of download_page
+  forEach ($release in $download_page) {
+    if (!$release.prerelease) {
```

Comment:
I want to ship pre-releases as well in the package. Please add a `-beta` to the version number to mark the Chocolatey package as a pre-release.

Reply:
I pushed a commit that will append `-beta` to the pre-release versions. This works so far, but I'm not sure what the upstream's vision is. They had a `WindowsTerminalPreview.msixbundle` and a `WindowsTerminal.msixbundle` in the stable release but only `WindowsTerminal.msixbundle` in the pre-release version.
What I'm curious is that the `WindowsTerminal.msixbundle` in the pre-release version actually installs the `Windows Terminal Preview`, a separate app than the `Windows Terminal`. Why would the upstream want to keep the same filename but different Get-Appx name?

Reply:
I don't know, unfortunately. We have to see what they do on the next release. I believe the previous release was both an Stable and a Preview release.